### ISSUE TITLE
[WARNING] STATE_ALARM_* deprecation in HA 2025.11

### ISSUE THREAD

**Issue by r0bb10** [petrleocompel/hikaxpro_hacs#136]
```Logger: homeassistant.const
Source: helpers/deprecation.py:222
First occurred: 16:02:09 (10 occurrences)
Last logged: 16:02:20

STATE_ALARM_ARMED_HOME was used from hikvision_axpro, this is a deprecated constant which will be removed in HA Core 2025.11. Use AlarmControlPanelState.ARMED_HOME instead, please report it to the author of the 'hikvision_axpro' custom integration
STATE_ALARM_ARMED_VACATION was used from hikvision_axpro, this is a deprecated constant which will be removed in HA Core 2025.11. Use AlarmControlPanelState.ARMED_VACATION instead, please report it to the author of the 'hikvision_axpro' custom integration
STATE_ALARM_ARMED_AWAY was used from hikvision_axpro, this is a deprecated constant which will be removed in HA Core 2025.11. Use AlarmControlPanelState.ARMED_AWAY instead, please report it to the author of the 'hikvision_axpro' custom integration
STATE_ALARM_DISARMED was used from hikvision_axpro, this is a deprecated constant which will be removed in HA Core 2025.11. Use AlarmControlPanelState.DISARMED instead, please report it to the author of the 'hikvision_axpro' custom integration
STATE_ALARM_TRIGGERED was used from hikvision_axpro, this is a deprecated constant which will be removed in HA Core 2025.11. Use AlarmControlPanelState.TRIGGERED instead, please report it to the author of the 'hikvision_axpro' custom integration
```

Reported today after upgrading to 2024.11 :)

**Comment by petrleocompel** [comment]
Will be fixed by #138

**Comment by petrleocompel** [comment]
Released as [v3.0.0-beta2](https://github.com/petrleocompel/hikaxpro_hacs/releases/tag/v3.0.0-beta2)

Later it will be upstreamed to "master" and pushed to "stable" release of 3.0.0.
